examples/decoding/plot_haxby_different_estimators.py

```diff
@@ -93,7 +95,8 @@
     print(70 * '_')
 
     for category in categories:
-        classification_target = stimuli[np.logical_not(resting_state)] == category
+        classification_target = stimuli[
+            np.logical_not(resting_state)] == category
```

Comment:
I already found this line hard to understand before (I particularly dislike playing with the priority of operators = and ==) but now the new_line make it even more obscure. Maybe we could change by:
```python
task_mask = np.logical_not(resting_state)  # or sample_mask?
classification_target = (stimuli[task_mask] == category)
```